dapr/clients/grpc/client.py

```diff
@@ -916,6 +917,40 @@ def get_configuration(
             items=items,
             headers=call.initial_metadata())
 
+    async def subscribe_configuration(
```

Comment:
I have a quick question ... Because of the async nature of the request this method is defined in the async await format ... But I find it odd that a Client class has all synchronous methods and the lone async method. .... 
cc @berndverst

Reply:
Yes this is odd, but also presents our first async grpc API

We probably need to support this eventually:
https://github.com/dapr/python-sdk/issues/90

Reply:
That is true ...